.github/actions/chaos-kitten/sarif-converter.py

```
@@ -0,0 +1,128 @@
+def generate_markdown_summary(stats, vulnerabilities, report_path):
+    """Generate Markdown summary for PR comment."""
+    summary = ["### 🐱 Chaos Kitten Security Scan Results"]
+    
+    # Summary table
+    summary.append("\n| Severity | Count | status |")
+    summary.append("|---|---|---|")
+    
+    icons = {
+        "critical": "🔴",
+        "high": "asd", # wait "asd"?? Typo prevention: "🟠"
+        "high": "🟠",
```

Ruff (0.14.14)

[error] Dictionary key literal `"high"` repeated

(F601)